docs/statistics.md

```diff
@@ -705,3 +718,86 @@ The value is used to apply TSBPD delay for reading the received data on the sock
+
+| Statistic                                       | Type of Statistic | Unit of Measurement | Available for Sender | Available for Receiver | Data Type |
+| ----------------------------------------------- | ----------------- | ------------------- | -------------------- | ---------------------- | --------- |
+| [msTimeStamp](#group-msTimeStamp)               | accumulated       | ms (milliseconds)   | ✓                    | ✓                      | int64_t   |
+| [pktSentUniqueTotal](#group-pktSentUniqueTotal) | accumulated       | packets             | ✓                    | -                      | int64_t   |
+| [pktRecvUniqueTotal](#group-pktRecvUniqueTotal) | accumulated       | packets             | -                    | ✓                      | int64_t   |
+| [pktRcvDropTotal](#group-pktRcvDropTotal)       | accumulated       | packets             | -                    | ✓                      | int32_t   |
+| [pktRcvDiscardTotal](#group-pktRcvDiscardTotal) | accumulated       | packets             | -                    | ✓                      | int32_t   |
+|                                                 |                   |                     |                      |                        |           |
+|                                                 |                   |                     |                      |                        |           |
+|                                                 |                   |                     |                      |                        |           |
+|                                                 |                   |                     |                      |                        |           |
+
+## Accumulated Statistics <a name="group-accumulated-statistics"></a>
+
+### msTimeStamp <a name="group-msTimeStamp"></a>
+
+The time elapsed, in milliseconds, since the time ("connection" time) when the initial group connection has been initiated (the time when the first connection in the group has been made and therefore made the group connected). This "connection" time will be then set in this statistic in every next socket that will become a member of the group as the new connections are established. A new connection to an already connected group doesn’t change the value of "connection" time. Available both for sender and receiver. 
+
+### pktSentUniqueTotal <a name="group-pktSentUniqueTotal"></a>
+
+The number of *unique original* DATA packets sent by the socket group. Available for sender.
+
+This value counts every *original* DATA packet sent over the network for the first time by the socket group. There is no difference between Connection Bonding modes (broadcast, backup and balancing). For example, sending the packet with a particular sequence number over multiple links in case of broadcast mode (it means sending this packet multiple times) does not affect the statistic and this very packet is taken into account only once.
```

Comment:
This description is actually wrong because it refers to mechanisms that are not involved in counting this metric.

This metric is counted as the number of packets that passed through the border between the application and the SRT library, when doing group sending, provided that the function used for sending (`srt_sendmsg2`) succeeded.

The simplest explanation for this metric is that this is the number of packets that the application has successfully scheduled for sending, that is, it has been successfully sent over any member connection. Whether the packet could be sent over the network, how many times and over how many links, the group sender has no available information.

Reply:
This requires further discussion on Monday then.

Reply:
Could you please write your description for this statistic?

Reply:
It's written in the last paragraph above.